components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1017,6 +1017,7 @@ struct BlockchainToken {
   string coingecko_id = "";
   string chain_id;
   CoinType coin;
+  bool is_shielded = false; // Zcash only, markes whether asset belongs to the shielded pool.
```

Comment:
marks

Reply:
also it makes sense to use `bool?` as this flag is optinal for other coins

Reply:
I guess we can't use optional primitives in mojom

Reply:
https://chromium.googlesource.com/chromium/src/+/lkgr/mojo/public/tools/bindings/README.md#primitive-types
It's been there for a while